app/assets/stylesheets/application.css.scss

```diff
@@ -10,6 +10,7 @@
 @import 'login';
 @import 'events';
 @import 'history';
+@import "parallax";
```

Comment:
I have to undo this, and add it back later. I had the correct parallax CSS elsewhere. Ill put it in a seperate file later, but this breaks the current parallax window.